client/js/components/user/orgaDataEntry/OrganisationDataForm.vue

```diff
@@ -322,16 +353,47 @@ export default {
     }
   },
 
+  data () {
+    return {
+      displayCustomer: hasPermission('feature_display_customer_names') && this.customers?.length > 0,
+      displaySlug: hasPermission('feature_orga_slug')
+        && !hasPermission('feature_orga_slug_edit')
+        && this.organisation.currentSlugName !== '',
+      hasPaperCopyPermission: hasPermission('field_organisation_paper_copy') ||
+        hasPermission('field_organisation_paper_copy_spec') ||
+        hasPermission('field_organisation_competence')
+    }
+  },
+
   computed: {
-    displaySlug () {
-      return hasPermission('feature_orga_slug') &&
-        !hasPermission('feature_orga_slug_edit') &&
-        this.organisation.currentSlugName !== ''
+    displayButtons () {
+      return this.isOrgaDataEditable
+        || this.hasPaperCopyPermission
+        || this.hasNotificationSection
+        || this.showDetailedInfo
+        || hasPermission('feature_change_submission_type')
     },
-
-    displayCustomer () {
-      return hasPermission('feature_display_customer_names') && this.customers?.length
+  },
+  methods: {
+    precheckSubmit (e) {
```

Comment:
This seems to not get called on submit (tested locally)

Reply:
Not at all? Or did you just don't run into the if?

Reply:
I set the breakpoint before the if and it didn't stop

Reply:
Fixed in c18f3b966f51b0ad650dc9b53d3c387f8d0bc088